### ISSUE TITLE
How the heck do you get a signature?

### ISSUE THREAD

**Issue by Bingies24** [pozm/gdke#22]
You tell us we need to get something but don't elaborate how to, effectively making the program useless without technical knowledge of Godot stuff that has no info online.

**Comment by TARNATlON** [comment]
Upvote

**Comment by BlackIce777f** [comment]
upvote

**Comment by char-ptr** [comment]
use ida or sigga to make a ida signature to the function in read me

**Comment by loldev69** [comment]
@char-ptr by the time you've found the function in the exe you can also find the encryption key yourself with minimal extra effort, far less effort than building a signature for the function.
A tool that you can't use until you no longer need it, peak Rust programming right here.

**Comment by bob-carter** [comment]
I agree, what even is the point of this software? Find the key, so you can find the key with your software... 

Should at least just brute force it instead.
